include/modem/lte_lc.h

```diff
@@ -286,6 +286,16 @@ struct lte_lc_cell {
 	 */
 	uint16_t timing_advance;
```

Comment:
The AT command specification has a new note for `timing_advance`:

_Note: Timing advance may be reported from past measurements. Client shall check the parameters <timing_advance_measurement_time> and serving cell <measurement_time> to evaluate if the param is usable._

Maybe this should be somehow be included in the API. I guess the idea is to calculate how old the timing advance measurement is and decide based on that whether it should be used or not.